src/buffer/stm_tests.ml

```diff
@@ -125,12 +124,13 @@ struct
     | _, _ -> false
 end
 
-module BufferSTM = STM.Make(BConf)
+module BufferSTM_Seq = STM.Seq.Make(BConf)
+module BufferSTM_Dom = STM.Domain.Make(BConf)
 
 ;;
 Util.set_ci_printing ()
 ;;
 QCheck_base_runner.run_tests_main
-  (let count = 1000 in
-   [BufferSTM.agree_test         ~count ~name:"STM Buffer test sequential";
-    BufferSTM.neg_agree_test_par ~count ~name:"STM Buffer test parallel" (* this test is expected to fail *)])
+  (let count = 100 in
+   [BufferSTM_Seq.agree_test     ~count ~name:"buffer test";         (* this test is expected to succeed *)
+    BufferSTM_Dom.agree_test_par ~count ~name:"buffer test parallel" (* this test is expected to fail *)])
```

Comment:
I think this is reverted from a negative test

Reply:
Yes, thanks for spotting this!